modules/salt/pages/formula-dhcpd.adoc

```diff
@@ -22,6 +22,10 @@ The DHCPd formula is used to configure the DHCP service on the branch server.
 . In the [guimenu]``Broadcast Address`` field, enter the broadcast IP address for the branch network (for example: [systemitem]``192.168.1.255``).
 . In the [guimenu]``Routers`` field, enter the IP address to be used by routers in the branch server network (for example: [systemitem]``192.168.1.5``).
 . In the [guimenu]``Next Server`` field, enter the hostname or IP address of the branch server (for example: [systemitem]``192.168.1.5``).
-. In the [guimenu]``Filename`` field, keep the default value of [systemitem]``/boot/pxelinux.0``.
+. In the [guimenu]``Filename`` field, if you are booting a client using PXE, type the path to the PXE bootloader.
+Otherwise, keep the default value of [systemitem]``/boot/pxelinux.0``.
+. In the [guimenu]``Filename Efi`` field, if you are booting a UEFI client using PXE, type the path the PXE bootloader.
+Otherwise, keep the default value of [systemitem]``/boot/shim.efi``.
```

Comment:
The default values will work for 90% of installations, so it is probably better to say
"If unsure, keep the default value ..."

Reply:
Heh, that feels like we're talking down to readers a bit (as in, "you probably won't understand this, it's very technical"). Does the "otherwise" not cut it?

Reply:
Maybe you can use different words, but it should be clear that the default values should not be changed in typical installation.